src/Routing/Loader.php

```diff
@@ -27,9 +27,11 @@ public function load($resource, $type = null) : RouteCollection
             $service->add('ping', "libero.content_api.{$config['name']}.ping");
 
             $service->add('items', "libero.content_api.{$config['name']}.item_list.get")
```

Comment:
as far as I understand, this then needs no additional configuration in the application :+1: